examples/example-generation/types/typelist

```diff
@@ -0,0 +1,873 @@
+ObjectDetector
```

Comment:
I'm not sure to be honest. I'm not familiar with this class. The typelist is simply scraped from the `.i` files in `src/interfaces/modular` as you can see in `example-generation/types/gettypelist`. 

Please let me know if I need to do something more smarter than that.